Tests/IndexComponent.test.js

```diff
@@ -0,0 +1,3 @@
+test('Two plus two is four', () => {
+  expect(2 + 2).toBe(4);
+});
```

Comment:
There should be an empty line at the end of file. Should set up linting to handle this.

Reply:
I've tried to look into next lint automatically running on 'next dev', but I've been unable to automate it. I've seen it at work, so I know it's possible.
I've timeboxed this issue and resolved it with 'eslint --fix', since all transgressions were mundane.